ml-image/scripts/evaluation_annotation.py

```diff
@@ -66,19 +72,18 @@ def label_results(detections: list, subframes: list, subframe_indices: list) ->
         face_centroid = ((x2 + x1) / 2, (y2 + y1) / 2)
         imgdraw.show()
 
-        # print(f'{ages[i]} ({np.round(age_confs[i], 2)}), {genders[i]} ({np.round(gender_confs[i], 2)}), {ethnicities[i]} ({np.round(ethn_confs[i], 2)})')
-        label = input(
-            'Do you want to label this face ? Answer with Yes or No.\n')
-        match label.lower():
+        annotation = input(
+            f'Do you want to annotate this {area_type} ? Answer with Yes or No.\n')
+        match annotation.lower():
             case 'yes':
                 gender = input(
```

Comment:
```suggestion
                gender = get_valid_input('Provide the perceived gender of the detected character (Male or Female).\n', ['Male', 'Female'])
```
ça peut peut être valoir le coup d'ajouter une méthode permettant d'obliger l'utilisateur à entrer une valeur reconnue par le modèle pour éviter d'avoir à tous réannoter en cas de typo. 
Exemple de méthode : 
````
def get_valid_input(prompt, valid_options):
    """
    Prompt the user for input and ensure it is valid.

    Args:
        prompt (str): The input prompt to display to the user.
        valid_options (list): A list of valid input options.

    Returns:
        str: The valid input provided by the user.
    """
    while True:
        user_input = input(prompt).strip()
        if user_input in valid_options:
            return user_input
        print(f"Invalid input. Please choose from: {', '.join(valid_options)}")
```